docs/getting_started.md

```diff
@@ -1,22 +1,76 @@
 # Getting Started
 
-To get started, install the requirements listed in the `requirements.txt` using a local python install or virtual env. Once installed, the model can be run using the `run_titan.py` program and configured using the `params.py` file.
+To get started, install titan using a local python (version 3.6 or later) install or virtual env using `pip`. Once installed, the model can be run using the `run_titan` program and configured using param files.
+
+```
+pip install titan-model
+run_titan -p my_params.yml
+run_titan -h
+```
 
 !!! tip
     Running a large job locally? Look into using [pypy](https://www.pypy.org/) instead of python for MOAR performance.  This is what we use on OSCAR.  Otherwise, all of the instructions hold, just using pypy and pypy's pip.
 
-## Installing the Dependencies
+## Development Setup
+
+Planning to help work on the titan codebase? Here are the tools and processes you need to know about:
+
+### Git/Github
+
+We use [git](https://git-scm.com/) for version control and GitHub for our remote repository. To get started, clone the repository to your local machine.
+
+```
+git clone https://github.com/pph-collective/TITAN.git
+```
+
+We use angular commits to standardize our commits and encourage better messages. [Commitizen](https://pypi.org/project/commitizen/) makes this easy.  Once installed (would recommend doing this globally as opposed to just for this project), run `cz commit` instead of `git commit`.
+
+### Poetry
+
+[Poetry](https://python-poetry.org/) is a python packaging and dependency management tool.  We use this to install/add/remove dependencies, build our package, and publish it to pypy.  Install poetry per [the install instructions](https://python-poetry.org/docs/#installation), then complete the below steps.
+
+```
+poetry install -E linting -E docs
+```
+
+The `-E` flags tell poetry that we want the optional dependencies that we use for linting (mypy, black, flake8) and for building documentation. *NOTE* black will not install on pypy, don't use pypy for running linting checks.  You may need to tell poetry which installation of python to use - see [their instructions](https://python-poetry.org/docs/managing-environments/).
```

Comment:
I think the comment on not using `black` with pypy is a bit confusing. It might be nice to more explicitly talk about pypy vs python with poetry?